logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/conversation/message/ProteusMessageUnpacker.kt

```diff
@@ -51,7 +51,7 @@ internal class ProteusMessageUnpackerImpl(
                 }
             }
             .map { PlainMessageBlob(it) }
-            .flatMap { plainMessageBlob -> getReadableMessageContent(plainMessageBlob, event.encryptedExternalContent) }
+            .flatMap { plainMessageBlob -> getReadableMessageContent(plainMessageBlob, event) }
```

Comment:
These are change not from you I guess, does the destination branch need to pull in develop?